src/forms/_checkbox.scss

```diff
@@ -0,0 +1,33 @@
+.checkbox {
+  display: none;
+
+  + .checkbox__label::before {
```

Comment:
We should decide if we prefer single or double colon notation for pseudo elements. Probably better to maintain consistency for now and just use one?